documentation/modules/oauth/con-oauth-authentication-flow.adoc

```diff
@@ -13,12 +14,25 @@ We recommend that you configure clients to use OAUTHBEARER whenever possible.
 OAUTHBEARER provides a higher level of security than PLAIN because client credentials are _never_ shared with Kafka brokers.
 Consider using PLAIN only with Kafka clients that do not support OAUTHBEARER.
 
-If necessary, OAUTHBEARER and PLAIN can be enabled together, on the same `oauth` listener.
+You configure Kafka broker listeners to use OAuth 2.0 authentication for clients connection.
+If necessary, you can enable OAUTHBEARER and PLAIN on the same `oauth` listener.
 
 .OAUTHBEARER overview
 
-Kafka supports the OAUTHBEARER authentication mechanism, however it must be explicitly configured.
-Also, many Kafka client tools use libraries that provide basic support for OAUTHBEARER at the protocol level.
+OAUTHBEARER is automatically enabled in the `oauth` listener configuration for the Kafka broker.
+You can set the `enableOauthBearer` property to `true`, though this is not required.
+
+[source,yaml,subs="attributes+"]
+----
+  # ...
+  authentication:
+    type: oauth
+    # ...
+    enableOauthBearer: true
+----
+
+The OAUTHBEARER authentication mechanism must be explicitly configured.
```

Comment:
Bit vague. I think we're talking about the actual _config_ rather than enabling. It applies to both mechanisms anyway, so I've moved up into the intro. I'll see what @mstruk thinks.

> You configure Kafka broker listeners to use OAuth 2.0 authentication for connecting clients.
> If necessary, you can use the OAUTHBEARER and PLAIN mechanisms on the same `oauth` listener.
> The properties to support each mechanism must be explicitly specified in the `oauth` listener configuration.